FreeRTOS/FreeRTOS-Kernel
# Style: Remove tabs and tab == 4 spaces

Style: Remove tabs and tab == 4 spaces from header files

Description
-----------
Some header files that do not contain tabs still have the 1 tab ==  4 spaces at the top
Some tabs are still in source files as uncrustify doesn't remove them, it was done manually

Test Steps
-----------
No source code has been modified

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
